src/components/ErrorComponent/ErrorComponent.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { FC, useEffect } from 'react';
+
+interface Props {
+  error: string;
+  onError: (error: string) => void;
+}
+
+export const ErrorComponent: FC<Props> = React.memo(({
+  error,
+  onError,
+}) => {
+  useEffect(() => {
+    const timeoutId = setTimeout(() => {
+      onError('');
+    }, 3000);
+
+    return clearTimeout(timeoutId);
```

Comment:
Please double check this part, you should return a function, but it seems you are returning a `clearTimeout` result